ops/model.py

```diff
@@ -60,9 +60,13 @@ def get_relation(self, relation_name, relation_id=None):
         """
         return self.relations._get_unique(relation_name, relation_id)
 
-    def get_binding(self, relation):
-        """Get a network space binding for a Relation object."""
-        return self._bindings.get(relation)
+    def get_binding(self, binding_key):
+        """Get a network space binding.
+
+        binding_key -- a name of a relation or a Relation object. Providing a Relation object will return
```

Comment:
I've usually used the Python rST docstring format:
https://www.python.org/dev/peps/pep-0287/

I have also seen things like "Google style doctrings":
https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

There doesn't seem to be One True Way:
https://stackoverflow.com/questions/3898572/what-is-the-standard-python-docstring-format

But I think we should look to standardize as a project, so that someone coming to the project doesn't have to reinterpret each command. It also will let us autogenerate some pretty html docs if we have a standard format.

Reply:
agreed we need to standardise. I don't think it should block this PR, as the whole codebase is a bit of a mismatch --- we should wrap up our conversation around this and land a single PR that Does The Thing.

Reply:
There was a discussion a while ago about that (I was suggesting this https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#info-field-lists to be used) but we never went along with that.

I think it might be a good time to revisit it for the whole code base but let's do the switch in a different PR when we decide on a particular approach.

Reply:
That would also be my preferred syntax (just from the syntaxes that I've used in the past).
I definitely agree that it is irrelevant for *this* patch, but I did want to remember for us to have the discussion.